### ISSUE TITLE
Transfer bug (?)

### ISSUE THREAD

**Issue by iRegal** [finndev/PokeBuddy#565]
So, in my "do not transfer" setting, i checked magikarp because i want to see the IV and transfer manually but why when i get two new magikarp, it still transfers the lower cp one ? although it still doesn't transfer magikarp i already have

**Comment by TeemoOnShrooms** [comment]
yeah selection is made on CP, if you don't want him to transfer a pkm just make it favourite (star)
